### DISCUSSION TITLE
c-store: A Column-oriented DBMS %E8%AE%BA%E6%96%87%E7%AC%94%E8%AE%B0 | Wu's Blog

### DISCUSSION THREAD

**Discussion by giscus** [gogim1/gogim1.github.io#6] [General]
# c-store: A Column-oriented DBMS %E8%AE%BA%E6%96%87%E7%AC%94%E8%AE%B0 | Wu's Blog

https://gogim1.github.io/posts/c_store

**Comment by nullptr-DZF** [comment]
谢谢，我从一篇SIGMOD压缩相关的论文找到C-Store并找到这里，非常有用
